src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -7530,37 +7515,54 @@ private BoundExpression MakeMemberAccessValue(BoundExpression expr, BindingDiagn
             }
         }
 
-        private BoundExpression ResolveToExtensionMemberIfPossible(BoundExpression expr, BindingDiagnosticBag diagnostics)
+#nullable enable
+        // When we're binding a member access that is not invoked and the member lookup yielded no result:
+        // - if an extension member lookup finds a non-method extension member (closer than any classic extension method compatible with the receiver
+        //   or than any method from an extension type), then that's the member being accessed.
+        //
+        // - if the extension member lookup finds a method (classic extension method compatible with the receiver or method in extension type;
+        //   closer than any non-method extension member), we don't need to touch the result for the member access (it's a method group already).
```

Comment:
>we don't need to touch the result for the member access (it's a method group already). [](http://example.com/codeflow?start=59&length=86)

It looks like this part of the comment is no longer relevant because we are no longer producing and passing in the method group.

Reply:
My intent is to add some documentation that relates to the callers (which are unfortunately split across two different methods). I'll try to clarify comment